### PR TITLE
jv-lottery

### PR DESCRIPTION
**Pull request by rafik2211**
create Ball class with the following fields: color and number and override toString() method. create enum with different colors. create a Lottery class with getRandomBall() method, which will return the ball. This method should create a ball with a random color and a random number and return this ball from the method.

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(Lottery.getRandomBall());
```
Comment (Ivan95kos) [inline src/main/java/core/basesyntax/Application.java:6]: create an instance of `Lottery`

File: src/main/java/core/basesyntax/Lottery.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100) + 1; // Random number between 1 and 100
+        return new Ball(randomColor, randomNumber);
+    }
```
Comment (Ivan95kos) [inline src/main/java/core/basesyntax/Lottery.java:10]: ColorSupplier and Random must be class fields
The method must not be a static
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
Remove the comments, code should be understandable without them

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```
Comment (Ivan95kos) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]: The method must not be a static
```suggestion
    public Color getRandomColor() {
```

**Review by Ivan95kos — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    private final Random random;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+        this.random = new Random();
+    }
+
+    public Ball getRandomBall() {
+        Color randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:16]: better, if 100 move to the class constant, because its magic number

**Review by kozhukhovsky — APPROVED** [review]

**Pull request closed without merging**